packages/contracts/contracts/optimistic-ethereum/ovm/ExecutionManager.sol

```diff
@@ -1147,12 +1146,12 @@ contract ExecutionManager is ContractResolver {
         return executionContext.l1MessageSender;
     }
 
-    function getCumulativeSequencedGas() public view returns(uint) {
-        return uint(StateManager(resolveStateManager()).getStorageView(METADATA_STORAGE_ADDRESS, CUMULATIVE_SEQUENCED_GAS_STORAGE_KEY));
+    function getCumulativeSequencedGas() public returns(uint) {
+        return uint(StateManager(resolveStateManager()).getStorage(METADATA_STORAGE_ADDRESS, CUMULATIVE_SEQUENCED_GAS_STORAGE_KEY));
```

Comment:
This seems to be used a bunch. Is it worthwhile to create a `getMetadataInt(key)` helper?